zebrad/src/components/mempool.rs

```diff
@@ -141,6 +148,14 @@ impl Service<Request> for Mempool {
                 let _ = self.storage.insert(tx);
             }
         }
+
+        let mut storage = self.storage.clone();
+        if let Some(tip_height) = self.latest_chain_tip.best_tip_height() {
+            let _ = tokio::task::spawn(async move {
+                remove_expired_transactions(&mut storage, tip_height).await
+            });
```

Comment:
Performance & Simplification Nitpick: Does `remove_expired_transactions` need to be async?

```suggestion
            remove_expired_transactions(&mut storage, tip_height);
```